src/Core/Components/MessageBar/FluentMessageBar.razor.cs

```diff
@@ -154,6 +159,13 @@ public DateTime? Timestamp
     [Parameter]
     public bool AllowDismiss { get; set; } = true;
 
+    /// <summary>
+    /// Gets or sets the fade in animation for the MessageBar.
+    /// Default is true.
+    /// </summary>
+    [Parameter]
+    public bool FadeIn { get; set; } = true;
```

Comment:
The default value must be `false` to respect the current result.

And to solve these unit tests

- ❌[FAILED] Microsoft.FluentUI.AspNetCore.Components.Tests.MessageBar.MessageServiceTests.FluentMessageBarService_AllowDismissAsync3
- ❌[FAILED] Microsoft.FluentUI.AspNetCore.Components.Tests.MessageBar.MessageServiceTests.FluentMessageBarService_AllowDismissAsync1
- ❌[FAILED] Microsoft.FluentUI.AspNetCore.Components.Tests.MessageBar.MessageServiceTests.FluentMessageBarService_AllowDismissAsync2

Reply:
I've pushed an update to those tests. But shouldn't the default stay `true` since the `MessageBar` in the current release has the fade-in animation?

Reply:
Yes, you're right.